README.markdown

```diff
@@ -59,8 +67,8 @@ instead call into uZFS process for serving "ioctls" using unix domain socket.
 Other than that the build steps are the same as for ZoL:
 
 ```bash
-git clone https://github.com/openebs/zfs.git
-cd zfs
+git clone https://github.com/openebs/cstor.git
+cd cstor
 ./autogen.sh
 CFLAGS="-g -O0" ./configure --enable-debug --enable-uzfs=yes
```

Comment:
`--enable-debug` option should be removed since it is not necessary for uZFS build.